apps/cnquery/cmd/builder/builder.go

```diff
@@ -54,7 +54,7 @@ func NewProviderCommand(opts CommandOpts) *cobra.Command {
 				os.Exit(1)
 			}
 
-			log.Info().Str("provider", "local").Msg("no provider specified, using defaults.\n  Use --help for a list of available providers.")
+			log.Info().Str("provider", "local").Msg("no provider specified, defaulting to local.\n  Use --help for a list of available providers.")
```

Comment:
If we add this to the text, we need to remove `.Str("provider", "local")` otherwise we add the same information twice